x-pack/plugins/ml/public/application/trained_models/models_management/test_models/models/text_classification/zero_shot_classification_input.tsx

```diff
@@ -18,33 +18,58 @@ import { INPUT_TYPE, RUNNING_STATE } from '../inference_base';
 const ClassNameInput: FC<{
   inferrer: ZeroShotClassificationInference;
 }> = ({ inferrer }) => {
-  const [labelsText, setLabelsText] = useState('');
-
-  useEffect(() => {
-    inferrer.labelsText$.next(labelsText);
-  }, [labelsText, inferrer]);
-
   const runningState = useObservable(inferrer.getRunningState$());
+  const labelsText = useObservable(inferrer.getLabelsText$());
+  const multiLabel = useObservable(inferrer.getMultiLabel$()) ?? false;
+
   return (
-    <EuiFormRow
-      fullWidth
-      label={i18n.translate(
-        'xpack.ml.trainedModels.testModelsFlyout.textClassification.classNamesInput',
-        {
-          defaultMessage: 'Class labels',
-        }
-      )}
-      helpText="Separate the labels with commas"
-    >
-      <EuiFieldText
-        value={labelsText}
-        disabled={runningState === RUNNING_STATE.RUNNING}
+    <>
+      <EuiFormRow
+        fullWidth
+        label={i18n.translate(
+          'xpack.ml.trainedModels.testModelsFlyout.textClassification.classNamesInput',
+          {
+            defaultMessage: 'Class labels',
+          }
+        )}
+        helpText={i18n.translate(
+          'xpack.ml.trainedModels.testModelsFlyout.textClassification.classNamesHelpText',
+          {
+            defaultMessage: 'Separate the labels with commas',
+          }
+        )}
+      >
+        <EuiFieldText
+          value={labelsText}
+          disabled={runningState === RUNNING_STATE.RUNNING}
+          fullWidth
+          onChange={(e) => inferrer.setLabelsText(e.target.value)}
+        />
+      </EuiFormRow>
+
+      <EuiSpacer size="m" />
+
+      <EuiFormRow
         fullWidth
-        onChange={(e) => {
-          setLabelsText(e.target.value);
-        }}
-      />
-    </EuiFormRow>
+        helpText={i18n.translate(
+          'xpack.ml.trainedModels.testModelsFlyout.textClassification.multiLabelHelpText',
+          {
+            defaultMessage: 'Is there a possibility of more than one correct label',
```

Comment:
This needs a `?` at the end of the sentence. Or alternatively from the docs `Indicates if more than one true label is possible given the input.`

Any thoughts @szabosteve ?

Reply:
What about:
`Enable the input text to have more than one label if possible.`

Reply:
Updated to `Enable the input text to match more than one label.`
https://github.com/elastic/kibana/pull/145357/commits/13a20c0d3393d0c50c488fa789af3c299b8ee2ef